test/backup_service/backup_service_test.go

```diff
@@ -161,6 +188,42 @@ var _ = Describe(
 				Expect(podList.Items[0].ObjectMeta.UID).ToNot(Equal(PodUID))
 			})
 
+			It("Should do hot-reload when dynamic fields are changed in backup service config", func() {
+				backupService, err = NewBackupService()
+				backupService.Spec.Service = &asdbv1beta1.Service{Type: corev1.ServiceTypeLoadBalancer}
+				Expect(err).ToNot(HaveOccurred())
+				err = DeployBackupService(k8sClient, backupService)
+				Expect(err).ToNot(HaveOccurred())
+
+				podList, gErr := getBackupServicePodList(k8sClient, backupService)
+				Expect(gErr).ToNot(HaveOccurred())
+				Expect(len(podList.Items)).To(Equal(1))
+
+				PodUID := podList.Items[0].ObjectMeta.UID
+
+				// Get backup service object
+				backupService, err = getBackupServiceObj(k8sClient, name, namespace)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Change dynamic fields")
+				// Remove all backup policies and storage entries
+				backupService.Spec.Config.Raw = []byte(`{"service":{"http":{"port":8081}}}`)
```

Comment:
why are we changing same field in static and dynamic config change test?

Reply:
We are changing the complete config in both.
In static we are changing the port and in dynamic we are keeping the same port.
Check the comment in the test-case